### DISCUSSION TITLE
Quick way: expand stroke to fill

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6727] [Ideas]
**Is your feature request related to a problem? Please describe**
for some cut-out workflow and vector building, many animators have a background experience with adobe animate, and that is to convert a stroke to a fill and make some work on it to create different shapes, there might not be a quick solution for this like there are on toonboom and adobe animate.

**Describe the solution you'd like**
a button near the line properties will fix the problem and upgrade workflow:
![expand to fill](https://user-images.githubusercontent.com/36406840/100485406-ac81e900-30de-11eb-914f-082213abf4ee.png)

**Comment by beeheemooth** [comment]
![ezgif-7-18fe32211f1f](https://user-images.githubusercontent.com/25724133/100515059-228b5c00-3182-11eb-99a9-e26a9bd748af.gif)

**Comment by age3rcm** [comment]
you could have helped me on the opentoonz discord heheh
nicevideo (gif) explaining!

is thhis on youtube?

**Comment by age3rcm** [comment]
i mean, the guy at discord wrote that there where many ways to do this and explained a way using bitmaps and level converters and...
